src/data/gods/hermes.ts

```diff
@@ -1,16 +1,17 @@
-import { COMMON, RARE, EPIC, LEGENDARY, HEROIC, DUO } from "./rarities";
-import { abilityFormatter } from "./formatters";
-import { ATTACK, SPECIAL, CAST, DASH, OTHER } from "./abilityTypes";
 import { mapValues, toArray } from "lodash";
+import { ATTACK, CAST, DASH, INFUSION, OTHER, SPECIAL } from "./abilityTypes";
+import { AIR, EARTH, FIRE } from "./elements";
+import { abilityFormatter } from "./formatters";
 import { Boon, God } from "./god";
-import { EARTH, AIR, } from "./elements";
+import { COMMON, EPIC, LEGENDARY, RARE } from "./rarities";
 
 const info =
   "Hermes, God of Speed and Commerce. His abilities increase your speed";
 
 const attack: Boon = {
   name: "Swift Strike",
   type: ATTACK,
```

Comment:
Hermes values are interesting; He clearly has an attack, special, and so on, but none of his boons occupy exclusive slots. I think we should still label them as-is, but I wonder if there is some way to highlight this. Maybe a simple [exclusive] / [unexclusive] tag at the end?

Reply:
I think we should move them all to be OTHER, since they don't use up slots. This made more sense to me in Hades 1 where his boons didn't use slots, but very closely mirrored the offerings from other gods. I'm not sure that passes muster in Hades 2 though.